scripts/plan_cost_runner.py

```diff
@@ -101,14 +127,31 @@ def print_diffs(diffs):
     for query_name, old_cost, new_cost in diffs:
         print("")
         print("Query:", query_name)
-        print("Old cost:", old_cost)
-        print("New cost:", new_cost)
+        print("Old total cost:", old_cost.total)
+        print("Old build cost:", old_cost.build_side)
+        print("Old probe cost:", old_cost.probe_side)
+        print("New total cost:", new_cost.total)
+        print("New build cost:", new_cost.build_side)
+        print("New probe cost:", new_cost.probe_side)
+
+
+def cardinality_is_higher(old_cost, new_cost):
+    new_cardinality_higher = (
```

Comment:
You could define a dunder method for this too, so you can directly compare `old_cost` and `new_cost`, keep the comparison logic closer to the definition of the class